yaus/sample.env

```diff
@@ -0,0 +1,2 @@
+NX_HASURA_GRAPHQL_ADMIN_SECRET=adminSuperSecret
+NX_HASURA_URL=http://localhost:15003/v1/graphql
```

Comment:
@radhay-samagra can you add the backend related env variables here as well?

@tushar5526 can you move the docker-compose file to this new folder.